BrickWall/models.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from flask import Flask, render_template, url_for, flash, redirect
+from flask_sqlalchemy import SQLAlchemy
+#from forms import RegistrationForm, LoginForm
+app = Flask(__name__)
+
+app.config['SECRET_KEY'] = 'thisIsATestSecretKey3'
+app.config['SQLALCHEMY_DATABASE_URL'] = 'sqlite:///site.db'
+db = SQLAlchemy(app)
+
+
+class User(db.model):
```

Comment:
We do not need a user model at all. We have Users through SSO and LDAP. We will have no need for this